lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosSensitivity.cxx

```diff
@@ -387,6 +388,43 @@ Scalar FunctionalChaosSensitivity::getSobolGroupedIndex(const Indices & variable
   else return 0.0;
 }
 
+
+
+/** Method that draw the sensitivity graph of a fixed marginal */
+Graph FunctionalChaosSensitivity::draw(UnsignedInteger marginalIndex) const
```

Comment:
This method should be merged with SobolIndicesAlgorithm::draw(). BTW, it was a really BAD idea to add a draw() method to an algorithm. An algorithm is supposed to produce a result, let's say a SobolIndicesResult, with drawing capabilities and so on. Maybe FunctionalChaosSensitivity should be renamed FunctionalChaosSobolIndices, with an inheritance wrt SobolIndicesResultImplementation. Something has to be refactored in details here.

Reply:
For now I only moved the getSobol* methods from FCRV to FunctionalChaosSensitivity.